client/extensions/woocommerce/app/settings/email/mailchimp/mailchimp_dashboard.js

```diff
@@ -0,0 +1,249 @@
+	const hasOrdersInfo = ( order_count !== undefined ) && ( mailchimp_total_orders !== undefined );
+	const orders = hasOrdersInfo ? ( order_count + '/' + mailchimp_total_orders ) : 'X/X';
+
+	const synced = () => (
+		<Notice
+			status="is-success"
+			isCompact={ true }
+			showDismiss={ false }
+			text={ syncState.mailchimp_list_name + ' ' + translate( 'list synced' ) }>
+		</Notice>
+	);
+
+	const syncing = () => (
+		<Notice
+			status="is-warning"
+			isCompact={ true }
+			showDismiss={ false }
+			text={ syncState.mailchimp_list_name + ' ' + translate( 'list is being synced' ) }>
+		</Notice>
+	);
+
+	const onResyncCLick = () => {
+		resync( siteId );
+	};
+
+	return (
+		<div>
+			<div>
+				<span className="mailchimp__account-info">{ translate( 'MailChimp account:' ) }</span>
+				<span>{ account_name }</span>
+			</div>
+			<span>{ store_syncing ? syncing() : synced() }</span>
+			<a onClick={ onResyncCLick }>Resync</a>
+			<div>
+				<span className="mailchimp__account-info" >{ translate( 'Products:' ) }</span>
+				<span>{ products }</span>
+				<span className="mailchimp__account-info" >{ ' ' + translate( 'Orders:' ) }</span>
+				<span>{ orders || '' }</span>
+			</div>
+		</div>
+	);
+} );
+
+const Settings = localize( ( { translate, settings, oldCheckbox, isSubbmittingSettings, onChange, onSave } ) => {
+	const onRadioChange = ( e ) => {
+		onChange( { mailchimp_checkbox_defaults: e.target.value } );
+	};
+
+	const onNewslatterLabelChange = ( e ) => {
+		onChange( { newsletter_label: e.target.value } );
+	};
+
+	const onToggle = ( e ) => {
+		const visibleOption = oldCheckbox !== 'hide' ? oldCheckbox : 'check';
+		onChange( { mailchimp_checkbox_defaults: e ? visibleOption : 'hide' } );
+	};
+
+	const checkbox = settings.mailchimp_checkbox_defaults;
+	const toggle = checkbox === 'check' || checkbox === 'uncheck';
+	return (
+		<div className="mailchimp__dashboard-settings">
+			<span className="mailchimp__dashboard-settings-form">
+				<FormFieldset>
+					<FormLegend>{ translate( 'Newsletter subscriptions' ) }</FormLegend>
+					<FormLabel>
+						<FormToggle
+							checked={ toggle }
+							onChange={ onToggle }
+							id="show-subscribe-message"
+						/>
+						<span>{ translate( 'Show a subscribe message to ustomer at checkout' ) }</span>
+					</FormLabel>
+					<FormLabel>
+						<FormRadio
+							disabled={ ! toggle }
+							value="check"
+							checked={ 'check' === checkbox }
+							onChange={ onRadioChange }
+						/>
+						<span>{ translate( 'Subscribe message is checked by default' ) }</span>
+					</FormLabel>
+					<FormLabel>
+						<FormRadio
+							disabled={ ! toggle }
+							value="uncheck"
+							checked={ 'uncheck' === checkbox }
+							onChange={ onRadioChange }
+						/>
+						<span>{ translate( 'Subscribe message is unchecked by default' ) }</span>
+					</FormLabel>
+					<FormLabel>
+						{ translate( 'Subscribe message is unchecked by default' ) }
```

Comment:
This string is repeated here again for the newsletter label. It should just be

'Subscribe Message'